package.json

```diff
@@ -8,6 +8,9 @@
     "eject": "expo eject"
   },
   "dependencies": {
+    "@react-native-community/netinfo": "^6.0.2",
```

Comment:
I don't see `@react-native-community/netinfo` or `aws-amplify-react-native` being used in this PR? I know we will need them eventually but as a best practice we should wait to add deps until they are needed

Reply:
That makes sense. I'll update it.